BeamlineStatusLogger/sources.py

```diff
@@ -194,3 +198,25 @@ def read(self):
             timestamp = datetime.now(self.localtz)
             value = None
         return Data(timestamp, value, metadata=metadata)
+
+
+class TINEAttributeSource:
+    def __init__(self, device_address, attr, metadata={}, tz=None):
+        if tine is None:
+            raise tine_import_err
+        self.device_address = device_address
+        self.attr = attr
+        self.metadata = metadata
+        if tz:
+            self.tzlocal = tz
+        else:
+            self.tzlocal = timezone('Europe/Berlin')
+
+    def read(self):
+        attribut = tine.get(self.device_address, self.attr)
+        test = attribut["data"]
```

Comment:
1. `test` is not a good variable name, maybe use `data` or `value`?
2. Before you access the `"data"` entry, you need to check the `"status"` entry (otherwise the `"data"` entry might not even exist).